src/services/api/features/package-actions/handlers/respond-to-rai-lambda.ts

```diff
@@ -0,0 +1,45 @@
+import { type APIGatewayEvent } from "aws-lambda";
+import { handleEvent } from "../lambda-handler";
+import { z } from "zod";
+import { PackageActionWriteService } from "../services/package-action-write-service";
+import { seatoolConnection } from "../consts/sql-connection";
+import { kafkaConfig } from "../consts/kafka-connection";
+import { response } from "@/libs/handler";
+import { TOPIC_NAME } from "../consts/kafka-topic-name";
+
+export const respondToRaiLambda = async (event: APIGatewayEvent) => {
+  return await handleEvent({
+    event,
+    schema: z.object({
+      id: z.string(),
+      latestRai: z.number(),
+      responseDate: z.number(),
+    }),
+    async lambda(data) {
+      const packageActionService =
+        await PackageActionWriteService.createPackageActionWriteService(
+          kafkaConfig,
+          seatoolConnection,
+          TOPIC_NAME
+        );
+
+      await packageActionService.respondToRai({
+        id: data.id,
+        latestRai: data.latestRai,
+        responseDate: data.responseDate,
```

Comment:
My assumption is that `data` is of shape `schema` as shown in the handle event config? If that's the case, can we just pass `data` here? Or did you find it more useful to have it declared like this?

Reply:
Yeah, I like that. Let me look into changing that